### PR TITLE
Sessions added to Global Search

### LINKED ISSUE #1826: Add Sessions to the searched results
**Actual Behaviour**
This doesn't exist now.

**Expected Behaviour**
This should be present within the search results

**Would you like to work on the issue?**
Yes

**Target Milestone to solve the issue**
Milestone 11

@arp95 @the-dagger @iamareebjamal @Shailesh351 Should we add only the bookmarked sessions or all the sessions? Adding all the sessions would clutter our search activity. What I can do is place the Bookmarked Sessions on the top above the Track Results. In another PR I can add the addition of the result filter i.e what all type of results does the user want.

### PR DESCRIPTION
**Pull request by shreyasnbhat**
Fixes #1826 

## Screenshot
9ef348ea-98e8-4555-a918-dab2a76c3e4d

### CODE REVIEW

**Comment by shreyasnbhat** [comment]
@iamareebjamal @Shailesh351 @arp95 @rohanagarwal94 Please review.

**Comment by shreyasnbhat** [comment]
@iamareebjamal @Shailesh351 @rohanagarwal94 Please review.

**Review by iamareebjamal — DISMISSED** [review]

File: android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java [hunk 1]
```diff
@@ -190,7 +192,7 @@ public void addResultsFromLocations(String queryString) {
 
         filteredMicrolocations.addChangeListener(microlocations -> {
             if(microlocations.size()>0) {
```
Comment (rohanagarwal94) [inline android/app/src/main/java/org/fossasia/openevent/activities/SearchActivity.java]: The formatting is not followed. There needs to be space after `if`. Its best to press `Ctrl + Alt + LShift + L` to format the entire file on the go.

**Review by rohanagarwal94 — CHANGES_REQUESTED** [review]

**Comment by shreyasnbhat** [comment]
@rohanagarwal94 @iamareebjamal @Shailesh351 Please check now. Updated.

File: android/app/src/main/java/org/fossasia/openevent/adapters/GlobalSearchAdapter.java [hunk 2]
```diff
@@ -63,7 +63,7 @@ public int getItemViewType(int position) {
         } else if (filteredResultList.get(position) instanceof Microlocation) {
             return LOCATION;
         } else if(filteredResultList.get(position) instanceof Session){
```
Comment (rohanagarwal94) [inline android/app/src/main/java/org/fossasia/openevent/adapters/GlobalSearchAdapter.java]: This file is not formatted. Please do it here also.

File: android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java [hunk 3]
```diff
@@ -238,7 +238,7 @@ private void loadData() {
                         if(!headerCheck){
```
Comment (rohanagarwal94) [inline android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java]: Even here.

**Review by rohanagarwal94 — CHANGES_REQUESTED** [review]

**Comment by shreyasnbhat** [comment]
@rohanagarwal94 @iamareebjamal @Shailesh351 Check now. Updated.

**Review by Shailesh351 — APPROVED** [review]
Looks good to me

**Review by rohanagarwal94 — APPROVED** [review]

**Comment by shreyasnbhat** [comment]
@iamareebjamal Please review.

**Comment by mariobehling** [comment]
Thank you.

**Pull request merged**